content/blog/architecture-as-code-intro/index.md

```diff
@@ -0,0 +1,307 @@
+---
+title: "Architecture as Code"
+date: 2020-03-18T13:42:35-05:00
+meta_desc: " An overview of fnfrastructure architecture and implementations with Pulumi"
+meta_image: architecture.png
```

Comment:
What are the objects in this image? I'm not sure I understand what the image is supposed to be communicating.

Reply:
Since the examples are in GCP, those are the logos for the GCP services. Amenable to change